Makefile

```diff
@@ -449,13 +449,25 @@ build/packages/docker/%/rhel-7:
 	docker rm docker-rhel7-$*
 
 build/packages/docker/18.09.8/rhel-8:
-	# unsupported
+	${MAKE} build/packages/docker/18.09.8/rhel-7-force
 
 build/packages/docker/19.03.4/rhel-8:
-	# unsupported
+	${MAKE} build/packages/docker/19.03.4//rhel-7-force
 
 build/packages/docker/19.03.10/rhel-8:
-	# unsupported
+	${MAKE} build/packages/docker/19.03.10/rhel-7-force
```

Comment:
isn't this the same thing as putting `build/packages/docker/19.03.10/rhel-7-force` after the colon on the line before it?

Reply:
hmmm, probably. lemme try that